android/src/ti/crashlytics/TitaniumCrashlyticsModule.java

```diff
@@ -63,6 +63,31 @@ public void throwException()
 		}
 	}
 
+	@Kroll.method
+	public void throwCustomException(String message, String myline, String sourceNameAndroid, String lineSource, String javascriptStack)
+	{
+		Log.d(message);
+		Throwable barT = new Throwable(message);
+		String[] lines = javascriptStack.split("\\r?\\n");
+		int count = 0;
+		StackTraceElement[] stackTarray = new StackTraceElement[lines.length];
+
+	   	for (String lineStack : lines) {
+
+		   	StackTraceElement stackNew = new StackTraceElement(sourceNameAndroid, lineStack, lineSource,  Integer.parseInt(myline));
+		   	stackTarray[count] = stackNew;
```

Comment:
if it is 0 all the time you don't need a variable `count`. But the question is: why only use the first element of `stackTarray` which you initialize with length `lines.length`. I think some count++ is missing somewhere. Again, just guessing at the moment since I've not used it yet